homework/index.js

```diff
@@ -29,23 +31,127 @@
     return elem;
   }
 
+  function addRow(table, title, repoText) {
+    const tr = createAndAppend('tr', table);
+    createAndAppend('th', tr, {
+      text: title,
+    });
+    createAndAppend('td', tr, { text: repoText });
+    return tr;
+  }
   function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+    const date = new Date(repo.updated_at);
+    const li = createAndAppend('li', ul, { class: 'li-root' });
+    const table = createAndAppend('table', li);
+    const firstRow = addRow(table, 'Repository:', '');
+    createAndAppend('a', firstRow.lastChild, {
+      href: repo.html_url,
+      text: repo.name,
+      target: '_blank',
+    });
+
+    addRow(table, 'Description:', repo.description, 'td');
+    addRow(table, 'Forks:', repo.forks, 'td');
+    addRow(table, 'Updated:', date.toLocaleDateString(), 'td');
+  }
+
+  function createOptions(repo, parent, index) {
+    createAndAppend('option', parent, {
+      text: repo.name,
+      value: index,
+    });
+  }
+  function selections(selection, data, parent) {
+    const ul = createAndAppend('ul', parent, {
+      class: 'ul-root',
+    });
+    const repo = data[selection];
+    renderRepoDetails(repo, ul);
+  }
+  function createContributorsSection(url, parent) {
+    fetchJSON(url).then(data => {
+      data.forEach(repo => {
```

Comment:
If you had used the correct name for the contributors array, you probably would not have made the mistake of using the name `repo` here instead of  `contributor`. Please make sure that you don't confuse other reading your code (or perhaps even yourself) by using incorrect names.